packages/addons/service/audio-tools/source/resources/language/English/strings.po

```diff
@@ -0,0 +1,44 @@
+# Kodi Media Center language file
+# Addon Name: service.audio-tools
+msgid ""
+msgstr ""
+
+msgctxt "#30000"
+msgid "Shared device"
+msgstr ""
+
+msgctxt "#30001"
+msgid "Select card"
+msgstr ""
+
+msgctxt "#30002"
+msgid "Card"
+msgstr ""
+
+msgctxt "#30003"
+msgid "Disable PulseAudio timer-based scheduling"
+msgstr ""
+
+msgctxt "#30004"
+msgid "Raspberry Pi onboard audio playback route"
+msgstr ""
+
+msgctxt "#30005"
+msgid "automatic"
+msgstr ""
+
+msgctxt "#30006"
+msgid "analogue"
```

Comment:
analog ? i know its AE and not BE but is `analogue` used at all  in BE ?

Reply:
I use the official terminology, from here https://www.raspberrypi.org/documentation/configuration/audio-config.md

Reply:
LE uses LibreEnglish ;)

Reply:
just asking as `analogue` is nothing I saw in the past at all ;D